app/roles/rhcos-images/tasks/main.yml

```diff
@@ -4,20 +4,33 @@
     state: directory
     mode: 0755
 
+- name: download rhcos image checksums
```

Comment:
Could we add a `changed_when: False` to this task?

Reply:
Sure, agree.